### PR TITLE
Move Sample Containers to Senaite setup folder

### PR DESCRIPTION
**Pull request by Bugerman58**
## Description of the issue/feature this PR addresses

Move SampleContainer and Sample Containers to Senaite setup folder, cleanup and refactoring

## Current behavior before PR

Now it lives in bika_setup folder

## Desired behavior after PR is merged

Now it lives in Senaite setup folder

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

**Comment by Bugerman58** [comment]
Hi, @ramonski this PR ready for review

File: src/senaite/core/upgrade/v02_06_000.py [hunk 1]
```diff
@@ -1564,6 +1564,42 @@ def move_instrumentlocations(tool):
...
+    tool.runImportStepFromProfile(profile, "workflow")
+
+    # get the old container
+    origin = api.get_setup().get("sample_containers")
+    if not origin:
+        # old container is already gone
+        return
+
+    # get the destination container
+    destination = get_setup_folder("sample_containers")
```
Comment (ramonski) [inline src/senaite/core/upgrade/v02_06_000.py]: Would you mind to change this to `samplecontainers` to match the naming scheme we used for the other setup folders?

Comment (Bugerman58) [reply src/senaite/core/upgrade/v02_06_000.py]: oh, sure. i was think about of renaming folder for this content type, but decided to wait for your consultation

Comment (ramonski) [reply src/senaite/core/upgrade/v02_06_000.py]: Thanks!

**Review by ramonski — APPROVED** [review]
Thanks!

**Pull request merged**